runtime-modules/proposals/codex/src/benchmarking.rs

```diff
@@ -951,12 +953,13 @@ mod tests {
         });
     }
 
-    #[test]
-    fn test_update_global_nft_limit_proposal() {
-        initial_test_ext().execute_with(|| {
-            assert_ok!(ProposalsCodex::test_benchmark_create_proposal_update_global_nft_limit());
-        })
-    }
+    // TODO: enable after Carthage
+    // #[test]
+    // fn test_update_global_nft_limit_proposal() {
+    //     initial_test_ext().execute_with(|| {
+    //         assert_ok!(test_benchmark_create_proposal_update_global_nft_limit::<Test>());
```

Comment:
The commented-out code is outdated
```suggestion
    //         assert_ok!(ProposalsCodex::test_benchmark_create_proposal_update_global_nft_limit());
```
(that's also the case for `test_update_channel_payouts_proposal`)